osu.Game.Rulesets.Catch/Edit/CatchEditorPlayfield.cs

```diff
@@ -23,5 +22,10 @@ protected override void LoadComplete()
 
             // TODO: disable hit lighting as well
         }
+
+        public void ApplyCircleSizeToCatcher(IBeatmapDifficultyInfo difficulty)
+        {
+            Catcher.SetScaleAndCatchWidth(difficulty);
+        }
```

Comment:
Method feels a bit pointless? Could just be inlined at the only point of usage (`Catcher` is internal so it might as well be public, pretty much, so you can do this from anywhere).